modules/common/vehicle_model/conf/vehicle_model_config.pb.txt

```diff
@@ -0,0 +1,5 @@
+model_type: REAR_CENTERED_KINEMATIC_BICYCLE_MODEL
+
+rc_kinematic_bicycle_model: {
+    dt: 0.06
```

Comment:
Why dt is 0.06?

Reply:
I found out the for 100ms prediction, dt, which decides the number of timestep (#timestep = 100ms /dt) only makes about 1e-9 error in x,y, and velocity when taking value from 0.0 to 100.0ms.

Reply:
by https://borrelli.me.berkeley.edu/pdfpub/IV_KinematicMPC_jason.pdf. The best dt for kinemetic model is around 200ms. As we don't predict 200ms, so a random number between 0.0 to 100ms is chosen since there isn't significant error anyway

Reply:
After adopting more models, dt will be an important conf to tune among models